onward/conf/routes

```diff
@@ -35,6 +35,7 @@ GET        /top-stories/trails.json                   controllers.TopStoriesCont
 GET        /related/*path.json                        controllers.RelatedController.render(path)
 
 GET        /popular-in-tag/*tag.json                  controllers.PopularInTag.render(tag)
+GET        /popular-in-tag-dcr/*tag.json              controllers.PopularInTagDCR.render(tag)
```

Comment:
We'll need to be adding these routes to router. Perhaps, as these are async calls going to `http://api.nextgen.guardianapps.co.uk/` we could have `/onwards/popular-in-tag` and then just route `location ~ ^/onward/.*\.json$` to the service?

Reply:
cc. @ioannakok -- might this relate to the issue with the `agents/` endpoint, too?